queries/dune_v2/period_slippage.sql

```diff
@@ -11,7 +11,12 @@ filtered_trades as (
            num_trades,
            solver_address,
            trader                                              as trader_in,
-           receiver                                            as trader_out,
+           -- Null Receiver: https://dune.com/queries/1729130
+           -- TODO - This could also be fixed in the spellbook!
```

Comment:
I will make PRs to Dune fixing this. so we no longer need this workaround.

Reply:
I would rename this field to `actual_receiver` or something similar (no not clobber the signed `receiver` value).